doc/source/_toc.yml

```diff
@@ -25,6 +25,7 @@ parts:
       sections:
         - file: ray-air/examples/torch_image_example
         - file: ray-air/examples/convert_existing_pytorch_code_to_ray_air
+        - file: ray-air/examples/convert_existing_tf_code_to_ray_air
```

Comment:
@justinvyu  this is great, just one general nit: can we drop the "convert_existing" prefix and just give the whole thing (the pytorch one, too) a name that describes the example? In a couple of weeks nobody will understand this anymore (what was it converted from).

Speaking of which, which doc was this converted from and can we remove the original now?

Reply:
The existing code is the quickstart from Pytorch/Tensorflow, and the idea is that someone can follow along to convert their own existing codebase to AIR. Does that need some more clarification in the beginning?